src/ResultUtils/Result.fs

```diff
@@ -1,8 +1,30 @@
+namespace ResultUtils.Portability
```

Comment:
I think 1 namespace per 1 file is more easy to understand.
Also, do you really need this namespace? it seems writing `open ResultUtils.Portability` in may files seems unnecessary change.

Reply:
Why don't we create another file and define the Result type in there, with the namespace `FSharp.Core.Result`?

Reply:
Using the namespace `FSharp.Core.Result` seems like a recipe for headaches since there's already an `FSharp.Core.Result` namespace with a `Result` type. It'll make it hard to disambiguate between the two `Result` types in the same namespace and to make sure you're using the one you intend to. And It'll make the code and the inevitable compilation errors a lot more confusing.

Reply:
> Using the namespace FSharp.Core.Result seems like a recipe for headaches since there's already an FSharp.Core.Result namespace with a Result type. It'll make it hard to disambiguate between the two Result types in the same namespace and to make sure you're using the one you intend to. And It'll make the code and the inevitable compilation errors a lot more confusing.

Right, this is fair point.

What about my first comment?

Reply:
I can move the namespace to another file easily enough. I'll do a PR to this PR for that.